src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/GroupModelRunParametersRouteConfig.kt

```diff
@@ -0,0 +1,33 @@
+package org.vaccineimpact.api.app.app_start.route_config
+
+import org.vaccineimpact.api.app.app_start.*
+import org.vaccineimpact.api.app.controllers.GroupModelRunParametersController
+import org.vaccineimpact.api.app.controllers.NewStyleOneTimeLinkController
+
+object GroupModelRunParametersRouteConfig : RouteConfig
+{
+    private val baseUrl = "/modelling-groups/:group-id/model-run-parameters/:touchstone-id"
+    private val controller = GroupModelRunParametersController::class
+
+    private val groupScope = "modelling-group:<group-id>"
+    private val permissions = setOf(
+            "$groupScope/estimates.write",
+            "$groupScope/responsibilities.read"
+    )
+
+    override val endpoints: List<EndpointDefinition> = listOf(
+            Endpoint("$baseUrl/", controller, "getModelRunParameterSets")
+                    .json()
+                    .secure(permissions),
+
+            Endpoint("$baseUrl/", controller, "addModelRunParameters")
+                    .post()
+                    .json()
+                    .secure(permissions),
+
+            Endpoint("$baseUrl/get_onetime_link/",
+                    NewStyleOneTimeLinkController::class, "getTokenForModelRunParameters")
```

Comment:
We actually don't need a onetime link for this any more, as am POST-ing the file from the portal using javascript, so able to include auth headers. But maybe removing this altogether is out of scope, as this is just a refactor. I'll raise a new ticket for it.

Reply:
Actually just realised when we refactor onetime links to use tokens, this endpoint will disappear anyway, so no need for a separate ticket